assets/bigquery/aggregate-tracks.sql.j2

```diff
@@ -1,112 +1,50 @@
 #standardSQL
```

Comment:
@andres-arana this is okay for a first approach but we need to review this query. Especially talking about the `{{ source }}*`.  For indo only this will consume 8.5GB per day, we need to do the math for AIS. We need to split this query to add incremental data. 

```
#standardSQL
WITH
  ########################################################
  # Extract the messages we are going to process
  ########################################################
  messages AS (
  SELECT
    sv.seg_id,
    sv.vessel_id,
    m.timestamp,
    ST_GEOGPOINT(m.lon, m.lat) AS position,
    IFNULL(m.nnet_score, m.logistic_score) AS score,
    m.speed,
    m.course
  FROM
    `pipe_indonesia_production_v20181029.messages_scored_*` AS m
  INNER JOIN
    `pipe_indonesia_production_v20181029.segment_info` AS si
  USING
    (seg_id)
  INNER JOIN
    `pipe_indonesia_production_v20181029.segment_vessel` AS sv
  USING
    (seg_id)
  WHERE
    si.noise = false
    AND si.pos_count > 5
    AND m.lat IS NOT NULL
    AND m.lon IS NOT NULL
    AND sv.vessel_id_rank = 1)
SELECT
  seg_id,
  vessel_id,
  timestamp,
  position,
  score,
  speed,
  course
FROM
  messages
```

Reply:
Just for reference, this is in a separate ticket at https://github.com/GlobalFishingWatch/GFW-Tasks/issues/1050

Reply:
Actually, I ended up implementing this here as well, since we need this for the indonesia backfill.